thinkhazard/static/less/report.less

```diff
@@ -249,6 +249,10 @@ footer {
   margin-right: 10px;
 }
 
+.hazard-active {
+    font-family: "Source Sans Pro Semibold";
```

Comment:
https://www.google.com/fonts/specimen/Source+Sans+Pro
OK, so you can use `font-weight: 600`